src/main/java/edu/cornell/kfs/module/purap/document/IWantDocument.java

```diff
@@ -97,6 +97,8 @@ public class IWantDocument extends FinancialSystemTransactionalDocumentBase impl
 
     private String completeOption;
     private boolean completed;
+    private String sscProcessorNetId;
+    private String sscProcessorName;
```

Comment:
I agree with a comment someone made in today's standup. I do not think that the term "ssc" should be associated in any part of the identifiers used in the coding nor in the database. That term can (and should) be in the label for the field where it can be easily modified in a single location when the names of departments change.  [SSC (shared service center) I believe used to be BSC (business service center) not too long ago.]

Would "processorNetId" / "processorName" OR "centralProcessorNetId" / "centralProcessorName" be more appropriate generic terms that could represent what these two fields stand for?